core/iwasm/common/wasm_runtime_common.c

```diff
@@ -7170,3 +7176,22 @@ wasm_runtime_detect_native_stack_overflow_size(WASMExecEnv *exec_env,
     }
     return true;
 }
+
+WASM_RUNTIME_API_EXTERN bool
+wasm_runtime_is_underlying_binary_freeable(const wasm_module_t module)
+{
+    bool is_freeable = false;
+
+#if WASM_ENABLE_INTERP != 0
```

Comment:
i'd suggest to make this return false for classic interpreter and lazy jit.

Reply:
classic interpreter will be enabled when JIT is enabled, maybe the code can be like:
```C
#if WASM_ENABLE_INTERP != 0
    if (module->module_type == Wasm_Module_Bytecode) {
#if (WASM_ENABLE_JIT != 0 || WASM_ENABLE_FAST_JIT != 0) && (WASM_ENABLE_LAZY_JIT !=0)
        is_freeable = false;
#elseif WASM_ENABLE_FAST_INTERP == 0
        is_freeable = false;
#else
        is_freeable = ((WASMModule *)module)->is_binary_freeable;
#endif
    }
#endif
```